### PR TITLE
Add ellswift to CHANGELOG

### PR DESCRIPTION
**Pull request by sipa**
A follow-up with a CHANGELOG entry for #1129.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+#### Added
+ - New module `ellswift` implements ElligatorSwift encoding for public keys and x-only Diffie-Hellman for them.
+   It permits representing secp256k1 public keys as 64-byte arrays which cannot be distinguished from uniformly random. See:
```
Comment (real-or-random) [inline CHANGELOG.md:12]: just editorial nits

```suggestion
 - New module `ellswift` implements ElligatorSwift encoding for public keys and x-only Diffie-Hellman key exchange for them.
   ElligatorSwift permits representing secp256k1 public keys as 64-byte arrays which cannot be distinguished from uniformly random. See:
```

**Comment by real-or-random** [comment]
Now that I see this nice list, it may also be good to add a reference to the .md in the API docs.

**Review by real-or-random — APPROVED** [review]
ACK c32ffd8d8c833a964ee7fbb294640764ad25de5d

**Comment by real-or-random** [comment]
> Now that I see this nice list, it may also be good to add a reference to the .md in the API docs.

Want to address this?

**Review by real-or-random — APPROVED** [review]
ACK 7c7467ab7f935f6b982064c8c48772a433da1f8f

**Review by theStack — APPROVED** [review]
ACK 7c7467ab7f935f6b982064c8c48772a433da1f8f

**Pull request merged**